### PR TITLE
Document registration of NutsComm endpoint on vendor and care organization DID documents

### PR DESCRIPTION
**Pull request by reinkrul**
Required for authenticated connections, but was missing.

### CODE REVIEW

File: mini-manuals/2-vendor-registration.md [hunk 1]
```diff
@@ -150,3 +150,26 @@ GET http://localhost:1323/internal/vdr/v1/did/{did}
...
+
+## Registering NutsComm endpoint
+
+To exchange private transactions (transactions that contain PII, personally identifiable information) your node's connections need to be authenticated with its DID document.
+This requires your node's `NutsComm` endpoint to be registered on the DID document.
+It is the endpoint that can be used by other nodes to connect to your node, and is required for authenticated connections.
+It must be in the format of `grpc://<host>:<port>`. You can register it through:
+
+```http request
+POST http://localhost:1323/internal/didman/v1/did/{did}/endpoint
```
Comment (pascaldekloe) [inline mini-manuals/2-vendor-registration.md:162]: What should be the `{did}` value in here?

Comment (pascaldekloe) [reply mini-manuals/2-vendor-registration.md:162]: Perhaps `{URN}` with:

You can register your DID by it's `"id"` with the following HTTP method.

Comment (woutslakhorst) [reply mini-manuals/2-vendor-registration.md:162]: `did` is quite a common term throughout the docs

**Review by woutslakhorst — APPROVED** [review]

File: mini-manuals/2-vendor-registration.md (hunk 1 above)
Comment (reinkrul) [reply mini-manuals/2-vendor-registration.md:162]: Being it part of the manual, DID is explained in an earlier section, I don't think it needs clarification here.

**Pull request merged**